src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/temporaryabsences/TemporaryAbsenceService.kt

```diff
@@ -29,4 +31,21 @@ class TemporaryAbsenceService(
       )
     }
   }
+
+  fun temporaryAbsencesArrival(
+    prisonNumber: String,
+    temporaryAbsencesArrival: TemporaryAbsencesArrival
+  ): InmateDetail {
+    return prisonApiClient.temporaryAbsencesArrival(
+      prisonNumber,
+      with(temporaryAbsencesArrival) {
```

Comment:
Looks like you are reusing the same model from the client - probably worth following the same pattern as transfers `TransferInDetail`and having a separate representation - it should have swagger annotations as it will be part of the contract when confirming temporary absences